src/main/java/me/totalfreedom/totalfreedommod/command/Command_opall.java

```diff
@@ -14,7 +14,7 @@
     @Override
```

Comment:
I'd suggest this command is tweaked. It's something we did on CJFreedom so it actually checks if you're an OP and if you were an OP before you never actually see the "You are OP Message" anyway.

Reply:
I'll keep that in mind, but I don't think I would be able to do that without fucking something up, @speedxx you wanna take that